DMTR-431.tex

```diff
@@ -40,8 +44,997 @@
+%mas\strut
+%\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
+%\(\leq 10\)\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%4.7\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%4.8\strut
+%\end{minipage} & \begin{minipage}[t]{0.17\columnwidth}\raggedright\strut
+%\strut
+%\end{minipage}\tabularnewline
+%\begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%AB1: \emph{i}\strut
+%\end{minipage} & \begin{minipage}[t]{0.06\columnwidth}\raggedright\strut
+%mas\strut
+%\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
+%\(\leq 10\)\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%5.7\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%6.0\strut
+%\end{minipage} & \begin{minipage}[t]{0.17\columnwidth}\raggedright\strut
+%\strut
+%\end{minipage}\tabularnewline
+%\begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%AB1: \emph{z}\strut
+%\end{minipage} & \begin{minipage}[t]{0.06\columnwidth}\raggedright\strut
+%mas\strut
+%\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
+%\(\leq 10\)\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%4.9\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%4.8\strut
+%\end{minipage} & \begin{minipage}[t]{0.17\columnwidth}\raggedright\strut
+%\strut
+%\end{minipage}\tabularnewline
+%\begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%AB1: \emph{y}\strut
+%\end{minipage} & \begin{minipage}[t]{0.06\columnwidth}\raggedright\strut
+%mas\strut
+%\end{minipage} & \begin{minipage}[t]{0.14\columnwidth}\raggedright\strut
+%\(\leq 10\)\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%7.0\strut
+%\end{minipage} & \begin{minipage}[t]{0.12\columnwidth}\raggedright\strut
+%7.2\strut
+%\end{minipage} & \begin{minipage}[t]{0.17\columnwidth}\raggedright\strut
+%\strut
+%\end{minipage}\tabularnewline
+\bottomrule
+\end{longtable}
+
+\section{Ellipticity Correlations}\label{ellipticity-correlations}
+
+\begin{figure}[!t]
+  \centering
+  \includegraphics[width=0.6\textwidth, trim=0.0in 0.0in 0.0in 0.0in, clip]{figures/ellip_metrics_v27_with_thresholds.png}
+  \caption{Median ellipticity residual correlations at 1-arcminute (TE1; normalized by a factor of $1\times10^{-5}$) and 5-arcminute (TE2; normalized by $1\times10^{-7}$) scales, as measured on $r$-band images, compared over the past few major pipelines releases. Measurements are compared against the SRD requirements (for both the ``design'' and ``minimum'' thresholds; note that the normalized minimum thresholds for TE1 and TE2 are the same, and thus indistinguishable on the figure). TE1 shows a slight improvement between v26 and v27, while TE2 increased significantly. \textbf{Q: Do we know what caused such a large jump in TE2?} }
```

Comment:
TE2 almost doubled in r and z band, showed a 25% increase in y, ~10% increase in g and  ~40% increase in i.  The increase varies greatly between bands - see attached quick 
<img width="612" alt="Screenshot 2024-08-02 at 17 20 51" src="https://github.com/user-attachments/assets/d2df0966-3024-4c5a-89eb-60070f88715b">
plot. It would be good to see get an explanation for this.

Reply:
If it can't be explained rapidly, then a jira ticket to investigate that is linked to this report